addon/data/shoot-panel.js

```diff
@@ -130,10 +130,8 @@ class SimplifiedPanel extends React.Component {
         <ShareButtons eventSource="simplified-panel" large={ true } { ...this.props } />
       </div>
       <div className="simplified-edit-container">
-        <button className="simplified-edit-button" onClick={ this.props.onClickEdit }>
           Edit Shot
         </button>
-        <button className="simplified-edit-button" onClick={ this.props.onClickAdd }>
```

Comment:
What's going on here?  This seems... wrong?  Also the images don't seem to be used?